src/CommunityToolkit.Maui.Camera/CameraManager.macios.cs

```diff
@@ -191,42 +196,44 @@ protected virtual async partial ValueTask PlatformTakePicture(CancellationToken
 
 		var capturePhotoSettings = AVCapturePhotoSettings.FromFormat(codecSettings);
 		capturePhotoSettings.FlashMode = photoOutput.SupportedFlashModes.Contains(flashMode) ? flashMode : photoOutput.SupportedFlashModes.First();
+		var photoOutputConnection = photoOutput.ConnectionFromMediaType(AVMediaTypes.Video.GetConstant()!);
+		photoOutputConnection!.VideoOrientation = videoOrientation;
```

Comment:
Can you please remove the null forgiving operator `!` and check for `null` instead.

Reply:
Done